### PR TITLE
Add Pokémon GO icon

### PR DESCRIPTION
**Pull request by KovuTheHusky**
**Issue: #1436**

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
SVG made in Illustrator and color is the classic Pokéball red.

### CODE REVIEW

**Comment by PeterShaggyNoble** [comment]
Thanks for picking this one up, @KovuTheHusky 👍 

Personally, I think an outline version similar to that in the original issue would be better. What do the other @simple-icons/maintainers think?

**Comment by ericcornelissen** [comment]
> Personally, I think an outline version similar to that in the original issue would be better. What do the other @simple-icons/maintainers think?

I'm actually more or less fine with both versions, though I do think something similar to the version in the issue is more recognizable out of context.

What I'm wondering more is whether this actually represents the Pokemon GO logo or Pokemon in general (or even just a Pokeball, which is not really a real world brand). If the brand here is Pokemon GO I would personally expect, maybe, the "GO" with a Pokeball inside the "O"?

More opinions on this last point from anyone? @KovuTheHusky @simple-icons/maintainers?

**Comment by phatbhoy67** [comment]
> I would personally expect, maybe, the "GO" with a Pokeball inside the "O"?

I agree with @ericcornelissen here.

"Pokemon" could be a future icon request and I'd assume in that case we would either use the wordmark or the "pokeball".

The "pokeball" seems to be used as a generic favicon across their sites, probably because the wordmark is deemed unsuitable? 🤔

**Comment by ericcornelissen** [comment]
> "Pokemon" could be a future icon request and I'd assume in that case we would either use the wordmark or the "pokeball".
>
> The "pokeball" seems to be used as a generic favicon across their sites, probably because the wordmark is deemed unsuitable? 🤔

That was, indeed, my reasoning a well.

**Comment by KovuTheHusky** [comment]
That's a fair point! How's this? Committed the SVG as well. Hope this is an improvement!

pokemongo

Is the color still appropriate or should it be changed? Here's the [official branding](https://upload.wikimedia.org/wikipedia/commons/2/23/Pok%C3%A9mon_GO_logo.svg) for reference.

File: _data/simple-icons.json [hunk 1]
```diff
@@ -3410,6 +3410,11 @@
             "hex": "F43E37",
             "source": "https://blog.pocketcasts.com/press/"
         },
+        {
+            "title": "Pokémon GO",
+            "hex": "FF0000",
+            "source": "https://www.pokemongo.com/"
```
Comment (ericcornelissen) [inline _data/simple-icons.json]: Given that the SVG now comes from somewhere else, can you update the source @KovuTheHusky?

```suggestion
            "source": "https://en.wikipedia.org/wiki/File:Pokemon_Go.png"
```

**Comment by KovuTheHusky** [comment]
@ericcornelissen Redid the icon and updated the source. Icon now includes the clip from the Earth's surface. Changes committed. I agree, it's an important element of the icon.

pokemongo

**Comment by ericcornelissen** [comment]
> Redid the icon and updated the source. Icon now includes the clip from the Earth's surface. Changes committed. I agree, it's an important element of the icon.

Thanks for the update, that version looks good to me but I will wait for an OK from another @simple-icons/maintainers before continuing 🙂 

That said, it seems you haven't committed the updated SVG just yet @KovuTheHusky?

**Comment by KovuTheHusky** [comment]
Pushed now - I swear I had but apparently not. Thanks!

**Comment by ericcornelissen** [comment]
@KovuTheHusky the lines in the [last update](https://github.com/simple-icons/simple-icons/pull/2340/commits/5e06dbcb25b0803a118e330700ed44ff89d10c67?short_path=e66c9af#diff-e66c9af94b8be776fd5bb1d32573f7d6) look slightly off 🤔 I don't have to tools to to a proper analysis now, can you (or a @simple-icons/maintainers) see what went wrong?

**Comment by KovuTheHusky** [comment]
I would agree with the deep blue used in the app icon and backgrounds, especially if red is used for Pokémon itself.

**Comment by PeterShaggyNoble** [comment]
Crap! Meant to comment on this earlier and now I'm on my phone so can't post screenshots. I zoomed in on this earlier and spotted the same problems Eric mentioned above, including gaps between the horizontal curve & the inner outlines of the letters and an overlap between the horizontal line in the Pokeball & the inner circle of it, maybe the outer one too - I can't remember now. Let me know if I'm not explaining those well enough and I'll post screenshots tomorrow.

It looks like the problems stem from the fact that the source SVG is a complete mess, which in turn seems to be because it's been automatically vectorised from a raster file

**Comment by ericcornelissen** [comment]
> Crap! Meant to comment on this earlier and now I'm on my phone so can't post screenshots. I zoomed in on this earlier and spotted the same problems Eric mentioned above, including gaps between the horizontal curve & the inner outlines of the letters and an overlap between the horizontal line in the Pokeball & the inner circle of it, maybe the outer one too - I can't remember now. Let me know if I'm not explaining those well enough and I'll post screenshots tomorrow.
>
> It looks like the problems stem from the fact that the source SVG is a complete mess, which in turn seems to be because it's been automatically vectorised from a raster file

Posting a screenshot would definitely help :smile:

**Comment by KovuTheHusky** [comment]
> Have any specific hex in mind (and a source for it) @KovuTheHusky?

I would assume the purest blue from the background in the app icon would be best. That would appear to be #0008a0.

**Comment by PeterShaggyNoble** [comment]
Well, that's weird: I'm no longer seeing the issue I described with the horizontal line of the Pokeball 😕

Screenshot below of on of the instances of gaps between the horizontal curve running through the letters and the letter outlines.

image

**Comment by ericcornelissen** [comment]
Going to close this now as abandoned, since there has been no activity on this Pull Requests since mid-February.

If anyone wants to contribute the Pokémon GO icon, do feel free to use this PR as a resource.

@KovuTheHusky if you still want to contribute the icon, feel free to update and reopen this PR :slightly_smiling_face:

**Comment by KovuTheHusky** [comment]
I would love to but I'm not sure how to fix the issue described, sorry!

**Pull request closed without merging**